src/components/EventList.js

```diff
@@ -19,10 +27,20 @@ const EventList = ({
   onRefresh,
   onPress
 }: Props) => (
-  <FlatList
+  <SectionList
     contentContainerStyle={styles.container}
-    data={events}
     keyExtractor={event => event.sys.id}
+    sections={events.map(it => ({
+      data: it,
+      title: format(parse(it[0].fields.startTime[locale]), "dddd D MMMM")
```

Comment:
I don't think `parse` is necessay here. You can just pass a string to `format`.

Reply:
Oh neat! I missed that. Convenient.